bears/dart/DartLintBear.py

```diff
@@ -2,6 +2,8 @@
 
 
 @linter(executable='dartanalyzer',
+        executable_check_fail_info='Please see the Installation section of page'
+                                   'https://github.com/dart-lang/linter.',
```

Comment:
eh i guess you're right, im going to change this as soon as my circleCI build finishes as i wanna see if it works.